kong/cmd/start.lua

```diff
@@ -18,7 +18,10 @@ local function execute(args)
   local dao = assert(DAOFactory.new(conf))
   xpcall(function()
     assert(prefix_handler.prepare_prefix(conf, args.nginx_conf))
-    assert(dao:run_migrations())
+    if args.run_migrations then
+      assert(dao:run_migrations())
+    end
+    assert(dao:are_migrations_uptodate())
```

Comment:
Also do we want this to be a fatal thing? Or should we just start the Kong and wait for the results of what happened to fan.